github/MainClass.py

```diff
@@ -775,43 +775,37 @@ def get_access_token(self, installation_id, user_id=None):
         body = None
         if user_id:
             body = json.dumps({"user_id": user_id})
-        conn = HTTPSConnection("api.github.com")
-        conn.request(
-            method="POST",
-            url="/installations/{}/access_tokens".format(installation_id),
-            headers={
-                "Authorization": "Bearer {}".format(self.create_jwt()),
-                "Accept": Consts.mediaTypeIntegrationPreview,
-                "User-Agent": "PyGithub/Python"
-            },
-            body=body
-        )
-        response = conn.getresponse()
-        response_text = response.read()
+        response = requests.post("https://api.github.com/installations/{}/access_tokens".format(installation_id),
```

Comment:
Why did you change this file?

Reply:
If you really want to change this, I would like to see them in a separate PR please

Reply:
Opened a new PR #998 for that. I will update this one.
It's good for codebase coherence I think.
Plus official doc says: 

> See also: The Requests package is recommended for a higher-level HTTP client interface. 

I'll update this PR.

Reply:
Awesome, will review